onnx/test/parser_test.py

```diff
@@ -218,6 +218,46 @@ def test_parse_node(self):
         self.assertEqual(node.domain, "SomeDomain")
         self.assertEqual(node.op_type, "SomeOp")
 
+    @parameterized.expand(
+        [
+            (
+                "not_a_good_float",
+                onnx.AttributeProto.GRAPH,
```

Comment:
With the change mentioned above, this example will produce a parse-error exception, which seems the correct behavior. You could either move this into a separate test, and check that a parse-exception is produced (or use a special value like None here and do it within this test).